config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -176,3 +176,76 @@ periodics:
                 --ignore-destroy-errors \
                 --powervs-memory 32 \
                 --test=ginkgo -- --parallel 10 --flake-attempts 2 --test-package-bucket k8s-release-dev --test-package-dir ci --test-package-version $K8S_BUILD_VERSION --focus-regex='\[Conformance\]' --skip-regex='\[Disruptive\]|\[Serial\]'
+  - name: periodic-kubernetes-containerd-node-throughput-perf-test-ppc64le
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    interval: 3h
+    extra_refs:
+      - base_ref: master
+        org: ppc64le-cloud
+        repo: kubetest2-plugins
+        workdir: true
+    spec:
+      volumes:
+        - name: powercloud-bot-key
+          secret:
+            defaultMode: 256
+            secretName: bot-ssh-secret
+      containers:
+        - image: quay.io/powercloud/all-in-one:0.3
+          command:
+            - /bin/bash
+          volumeMounts:
+            - mountPath: /etc/secret-volume
+              name: powercloud-bot-key
+              readOnly: true
+          envFrom:
+            - secretRef:
+                name: ibm-cloud-credentials
+          args:
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              export PATH=$GOPATH/bin:$PATH
+              export GO111MODULE=on
+
+              go install ./...
+
+              go get sigs.k8s.io/kubetest2@latest
+              go get sigs.k8s.io/kubetest2/kubetest2-tester-clusterloader2@latest
+
+              TIMESTAMP=$(date +%s)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+              jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > $ARTIFACTS/metadata.json
+
+              # kubectl needed for the e2e tests
+              curl -sSL https://dl.k8s.io/ci/$K8S_BUILD_VERSION/bin/linux/`go env GOARCH`/kubectl > /usr/local/bin/kubectl
+              chmod +x /usr/local/bin/kubectl
+
+              pushd /tmp;git clone https://github.com/kubernetes/perf-tests.git
+              popd
+
+              kubetest2 tf --powervs-dns k8s-tests \
+                --powervs-image-name centos-83-12142020 \
```

Comment:
we already have an image in the tier1 with name `centos-83-03222021-tier1`, you can mentioned that here